Add IDB open data catalog to list of trusted repositories

branmora opened the pull request:
Person Updating Information: Brandon Mora (bmoraperez@iadb.org)
Date: 2025-06-30

branmora committed:
add idb open data catalog

larsvilhuber reviewed (changes requested):
@branmora Could you clarify a few things:

- Right now, I only see data, but not code, and it isn't clear to me how code would be presented.
- As part of your pull request, you linked to the policy document about access; however, what we are interested in is the policy on preservation and retraction of published objects. In other words, what is the stated policy on how long items are kept (ideally, "forever", but it must be an explicit statement), and how once-published objects can be withdrawn (should be "never", but there should be a policy when items contain inappropriate objects, etc.). 
- You have noted that "access" is "all", but I don't believe that is true. Access here is meant for "who can deposit" (we might make that cleaner). 
- You said "yes" to whether there is a re3data entry. Most entries have "No", but if you have a re3data entry, you should provide the link to it here. Some of the elements that we are asking here are usually available within a re3data entry.

branmora committed:
Fix: Address reviewer feedback

branmora commented:
@larsvilhuber here are some comments:

1. Currently, our replications can be found under the research data bucket. Also accessible using this url: https://data.iadb.org/dataset/?q=Replication. We provide all replication-related files in the "Additional materials" section (.do, .R, ,py files, additional docs, among others)
2. We've updated the URL to the bank's relevant guidelines about preservation and retraction: https://www.iadb.org/document.cfm?id=EZIDB0000576-908968382-863. While the guidelines do not explicitly state “forever” or “never” regarding preservation or retraction, it provides relevant guidance:
  - Retention: Annex I, Section A notes that public documents will not be removed unless aligned with formal retention schedules, implying long-term availability.
  - Amendments: Section 4.14 states that updated documents replace the old, but prior versions remain public and available upon request.
  - Corrections: Section 4.7.c allows for replublishing/removing in case of errors or inconsistencies.
3. We've updated the access field, only IDB Employees can publish data on the repository.
4. The re3data entry is now included in the field.

larsvilhuber reviewed (approved):
Seems fine now. I will add myself as Recommender.
